r/R/dplyr-across.R

```diff
@@ -23,7 +23,7 @@ expand_across <- function(.data, quos_in) {
     quo_expr <- quo_get_expr(quo_in[[1]])
     quo_env <- quo_get_env(quo_in[[1]])
 
-    if (is_call(quo_expr, "across")) {
+    if (is_call(quo_expr, c("across", "if_any", "if_all"))) {
```

Comment:
Be sure to update docgen.R to reflect what is supported in `across()` now. You may also want to add entries there for `if_any` and `if_all` (they're tidyselect?) since they're supported too but not automatically included in the docs since they don't go through `register_binding`

Reply:
Done!